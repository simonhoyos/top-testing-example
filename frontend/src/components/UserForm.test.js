import React from 'react';
import { render } from '@testing-library/react';
import { UserForm } from './UserForm';

describe('UserForm', () => {
  it('should render correctly', () => {
    const { container } = render(<UserForm />);

    expect(container.firstChild).toMatchSnapshot();
  });
})
