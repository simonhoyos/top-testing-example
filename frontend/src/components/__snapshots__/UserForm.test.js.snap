// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserForm should render correctly 1`] = `
<form>
  <label
    for="email"
  >
    Email
  </label>
  <input
    id="email"
    name="email"
    type="text"
    value=""
  />
  <label
    for="password"
  >
    Password
  </label>
  <input
    id="password"
    name="password"
    type="password"
    value=""
  />
  <button>
    Submit
  </button>
</form>
`;
